src/content/docs/installation/method/uninstall-unix.mdx

```diff
@@ -17,9 +17,15 @@ __Prerequisites__
 
 __Procedure__
 
-1. Run the following commands to purge local Daytona data and remove the binary:
+1. Run the following command to stop the Daytona Server(if it is running):
+
+    ```shell
+    daytona server stop
+   ```
+
+2. Run the following commands to purge local Daytona data and remove the binary:
 
     ```shell
     daytona purge
-    sudo rm $(where daytona)
```

Comment:
By the way `where` command is for Windows while The Linux/Unix/Mac equivalent command is `whereis` 
Read from here
[where](https://learn.microsoft.com/en-us/windows-server/administration/windows-commands/where)
[whereis](https://linuxize.com/post/whereis-command-in-linux/)

Reply:
`where` works on linux and mac as well.